lib/index.js

```diff
@@ -111,8 +111,8 @@ function createServiceAPI(serviceName) {
         if (serviceName === 'search') {
           throw new Error('The search service has been replaced by retrieve_and_rank');
         }
-        if (serviceName === 'tone_analyzer' && path.basename(version) === 'v1') {
-          throw new Error('tone_analyzer v1 has been replaced by v2-experimental');
+        if (serviceName === 'tone_analyzer' && path.basename(version) === 'v3-beta') {
+          throw new Error('tone_analyzer v3-beta has been replaced by v3');
         }
 
         // for the v1.0 breaking change of requiring experimental/beta in the module name, try to offer a helpful error message.
```

Comment:
We should probably make a more general version of this for any time the specified version isn't found. But this is good enough for today.